docs/configuration.md

```diff
@@ -589,6 +589,81 @@ foo // c1
 }
 ```
 
+#### `indent.fewerBraces`
+
+This parameter controls whether extra `indent.main` is added
+to the sole argument of a method call using the "fewer braces"
+syntax. The following values are supported:
+
+- `always`: always applies extra indent
+- `never`: doesn't apply any extra indent
+- `beforeSelect`: applies extra indent only to fewer-braces
+  expressions followed by a `.select`
+
+```scala mdoc:defaults
+indent.fewerBraces
+```
+
+> In Scala 3.3.0-RC3, only `never` provides compiler-compatible code.
```

Comment:
Should we introduce a setting based on something that might be fixed? I would hold off in that case

Reply:
could you explain it, please? if this is fixed in the compiler, some people might still prefer different options. and if it's not, then the current implementation doesn't work.

by the way, are you and Martin close to an agreement on that fix? :)

Reply:
`only `never` provides compiler-compatible code.`

That's what worries me, if user sets something wrong it breaks their compilation, no?

> by the way, are you and Martin close to an agreement on that fix? :)

I need to come back to it, don't remember what's the current status :O

Reply:
the only other suggestion i might have is

- add a new dialect field in scalameta to say select can be indented after fewer braces
- set that flag in scala33
- add a new intermediate scala330rc2 dialect where this flag is not set
- move fewer braces to that dialect as well, as it hadn't been available before that.

Reply:
> so what's the solution, then? right now, it will always break their compilation...

Couldn't we make `never` the default without the new setting for now?

Reply:
> > so what's the solution, then? right now, it will always break their compilation...
> 
> Couldn't we make `never` the default without the new setting for now?

done